modules/java_api/org/intel/openvino/PrePostProcessor.java

```diff
@@ -13,16 +22,35 @@ public PrePostProcessor(Model model) {
         super(GetPrePostProcessor(model.nativeObj));
     }
 
+    /**
+     * Gets input pre-processing data structure. Should be used only if model/function has only one
+     * input Using returned structure application's code is able to set user's tensor data (e.g
+     * layout), preprocess steps, target model's data
+     *
+     * @return Reference to model's input information structure
+     */
     public InputInfo input() {
         return new InputInfo(Input(nativeObj));
     }
 
+    /**
+     * Gets output post-processing data structure. Should be used only if model/function has only
+     * one output Using returned structure application's code is able to set model's output data,
+     * post-process steps, user's tensor data (e.g layout)
+     *
+     * @return Reference to model's output information structure
+     */
     public OutputInfo output() {
         return new OutputInfo(Output(nativeObj));
     }
 
-    public void build() {
-        Build(nativeObj);
+    /**
+     * Adds pre/post-processing operations to function passed in constructor
+     *
+     * @return Function with added pre/post-processing operations
+     */
+    public Model build() {
```

Comment:
then native method `Build` can't be `void`

Reply:
Thanks! Done